Common.mk

```diff
@@ -12,47 +12,58 @@
 # limitations under the License.
 
 SHELL := /bin/bash
+
+AUTO_BUILD_VERSION ?= dev
+OURDIR := BUILD
+OUTPUT := ${OURDIR}/$(shell basename $(shell dirname $(shell realpath $(firstword $(MAKEFILE_LIST)))))
```

Comment:
Can't this be simplified to `OUTPUT := ${OURDIR}/$(shell basename $(pwd))`?

Reply:
fixed